src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationNotesTest.kt

```diff
@@ -553,3 +553,542 @@ class Cas2ApplicationNotesTest(
     }
   }
 }
+
+class Cas2AssessmentNotesTest(
```

Comment:
I think it's ok in this file for now, since when we get to removing the legacy code after the front end has been updated `Cas2AssessmentNotesTest` will be the only tests in this file we can rename it to `Cas2AssessmentNotesTest.kt`.  We will be removing the `Cas2AssessmentNotesTest` class which test the (to be) legacy endpoint `submissions/$applicationId/notes`.